app/src/main/res/layout/navigation_bar_navigation.xml

```diff
@@ -11,11 +11,20 @@
             name="settingsmodel"
             type="com.igalia.wolvic.ui.viewmodel.SettingsViewModel" />
     </data>
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        android:gravity="center_vertical"
```

Comment:
`android:gravity="top"` might work better in this case, so the buttons will always be aligned to the top of the navigation bar layer.

Reply:
it does not really matter much because that LinearLayout is not the navigation bar, is the container that wrap both the navigation bar and the move bar. The navigation bar buttons are already aligned.